app/src/main/java/com/opsat7373/moviestars/data/datasource/WebDataSource.kt

```diff
@@ -0,0 +1,39 @@
+package com.opsat7373.moviestars.data.datasource
+
+import androidx.lifecycle.MutableLiveData
+import com.opsat7373.moviestars.data.model.Movie
+import com.opsat7373.moviestars.data.model.PopularMoviesList
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.*
+
+
+class WebDataSource : MovieDataSourceInterface {
+    override fun getAll(): MutableLiveData<List<Movie?>> {
```

Comment:
If you do not modify the list, it's better to expose just LiveData, not Mutable

Reply:
I am going modify this list in another methods. Also live data it's abstract class